tests/test_admin_views.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+
+from django.http import Http404
+
+from accession import admin_views
+
+from .factories import DonorFactory, AccessionFactory
+
+pytestmark = pytest.mark.django_db
+
+
+class TestDuplicates():
+
+    def test_returns_200_with_mapped_model(self, rf, admin_user):
+        request = rf.get('/admin/accession/donor/duplicates/')
+        request.user = admin_user
+
+        response = admin_views.duplicates(request, 'donor')
+
+        assert response.status_code == 200
+
+    def test_returns_200_with_unmapped_model(self, rf, admin_user):
+        request = rf.get('/admin/accession/city/duplicates/')
+        request.user = admin_user
+
+        response = admin_views.duplicates(request, 'city')
+
+        assert response.status_code == 200
+
+    @pytest.mark.xfail(reason='The view does not catch the exception raised'
```

Comment:
Hmmmm, I'm actually beginning to think that this test should actually be a test for the query_to_tuple() function. What do you think?